engine/src/main/java/org/entando/entando/aps/system/services/api/Unmarshaller.java

```diff
@@ -0,0 +1,38 @@
+package org.entando.entando.aps.system.services.api;
+
+import com.agiletec.aps.system.SystemConstants;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.io.IOException;
+import java.io.InputStream;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+
+@Component(SystemConstants.UNMARSHALLER)
+public class Unmarshaller {
```

Comment:
the name is too much generic in particular for a bean deeply coupled with legacy API  implementation, can we change it?

Reply:
Named `LegacyApiUnmarshaller`. Also renamed the constant.